time/time.c

```diff
@@ -158,7 +160,7 @@ double difftime(time_t t1, time_t t2)
 struct tm *gmtime_r(const time_t *timep, struct tm *res)
 {
 	time_t sec = *timep, minute = 0, hou = 0, t;
-	int day = 0, leap, leapcnt;
+	int day = 0, leap, leapcnt, full_400_years;
```

Comment:
Avoid snake_case, use camelCase
```suggestion
	int day = 0, leap, leapcnt, full400years;
```